contributed_definitions/nyaml/NXcalibration.yaml

```diff
@@ -49,13 +60,38 @@ NXcalibration(NXobject):
     unit: NX_ANY
     doc: |
       A vector representing the axis after calibration, matching the data length
+    \@output_path:
+      doc: |
+        The path to which this data is written, e.g., the calibrated energy.
+        Should be a valid NeXus path name, e.g., /entry/data/energy.
     dimensions:
       rank: 1
       dim: [[1, ncal]]
   original_axis(NX_FLOAT):
```

Comment:
Maybe we can restructure a bit, such that we first have the inputs, then the fit_function and coefficients, and then the results. Just for readability.